docs/src/types/jump_types.md

```diff
@@ -134,3 +191,15 @@ JumpProblem(prob,Direct(),jump1,jump2)
 
 will build a problem where the constant rate jumps are solved using Gillespie's
 Direct SSA method.
+
+
+## Recommendations for Constant Rate Jumps
+For representing and aggregating constant rate jumps 
+- Use a `MassActionJump` to handle all jumps that can be represented as mass
+  action reactions. This will generally offer the fastest performance and fewest
```

Comment:
Do any of the aggregators require heap allocations in the inner loop? I don't think they do?

Reply:
OK.